prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -1984,4 +1985,95 @@ class OtcProctored : CoolTest() {
             return bad("Tests FAILED: " + failures)
         }
     }
+}
+
+class CrudRestApi : CoolTest() {
+    override val name = "CrudRestApi"
```

Comment:
all other tests use lowercase name.  All our APIs are CRUD REST APIs, so this name isn't illuminating.

How about naming it 'settings' , since that's what it tests?